CMakeLists.txt

```
@@ -455,15 +455,15 @@ set_option(SDL_ALSA                "Support the ALSA audio API" ${UNIX_SYS})
 dep_option(SDL_ALSA_SHARED         "Dynamically load ALSA audio support" ON "SDL_ALSA" OFF)
 set_option(SDL_JACK                "Support the JACK audio API" ${UNIX_SYS})
 dep_option(SDL_JACK_SHARED         "Dynamically load JACK audio support" ON "SDL_JACK" OFF)
-set_option(SDL_ESD                 "Support the Enlightened Sound Daemon" ${UNIX_SYS})
+set_option(SDL_ESD                 "Support the Enlightened Sound Daemon" OFF ${UNIX_SYS} OFF)
```

`set_option` is an unconditional option.
`dep_option` is a conditional option.
```suggestion
det_option(SDL_ESD                 "Support the Enlightened Sound Daemon" OFF ${UNIX_SYS} OFF)
```

---

Done - I hope, please confirm.

---

Looks okay now!